Gruntfile.js

```diff
@@ -23,8 +23,24 @@ module.exports = function (grunt) {
   var appConfig = {
     app: require('./bower.json').appPath || 'app',
     dist: 'client/dist'
+  }
+    // Configurable paths for the application
+  var bowerJson = require('./bower.json');
+  var bowerRc;
+  try {
+      bowerRc = JSON.parse(fs.readFileSync('./.bowerrc'), 'utf-8');
+  } catch(err) {
+      // No .bowerrc.  Not a problem--it's optional!
+  }
+  var appConfig = {
+      app: (bowerJson && bowerJson.appPath) ? bowerJson.appPath : 'client/ngapp',
+      // vendor: (bowerRc && bowerRc.directory) ? bowerRc.directory : 'client/ngapp/vendor',
+      vendor: (bowerRc && bowerRc.directory) ? bowerRc.directory : 'bower_components',
+      dist: 'client/dist',
+      stage: '.tmp'
```

Comment:
Looks like your knowledge is better than mine, I'll trust your judgment here.

My concern is to reduce duplication of values. If we know the value is already set in `.bowerrc`, then we should not repeat it in `Gruntfile.js` as a default value. I don't really mind where the value is defined, as long as there is only one place where it lives.

> That said, editing the proposal to presume the file will exist means I can drop the ternary operator use and leave the fs.readFileSync() call unprotected by try/catch. Those two items alone will yield a much smaller footprint.

Sounds good :+1: